js/components/urlBarSuggestions.js

```diff
@@ -379,6 +379,17 @@ class UrlBarSuggestions extends ImmutableComponent {
       }))
     }
 
+    // Local search suggestions
+    suggestions = suggestions.concat(mapListToElements({
+      data: props.localSearchTerms.map((e) => { return e.get('searchTerm') }),
```

Comment:
Why not use `localSearchTerms.map(e => e.get('searchTerm'))` which is more readable?